src/workerd/api/sql.h

```diff
@@ -249,10 +250,38 @@ class SqlStorage::Statement final: public jsg::Object {
   friend class Cursor;
 };
 
+
+
+class SqlStorage::IngestResult final : public jsg::Object {
+public:
```

Comment:
Does this really need to be a `jsg::Object`? Could `IngestResult` just be a `JSG_STRUCT`?

Reply:
Sure! I only went with `jsg::Object` since that's what the other classes in this header file inherit from. I've never heard of JSG_STRUCT, but I can take a look at using it.

Reply:
The key difference between the two is that when a `JSG_STRUCT` type is passed out to JavaScript, it comes in the form of an ordinary JavaScript object with properties as opposed to an instance of a class. There is also no persistent c++ object behind the object. The struct is thrown away as soon as the JavaScript object is created.